src/components/date-picker-day/date-picker-day.scss

```diff
@@ -93,25 +93,15 @@
   }
 }
 
-:host(:focus),
-:host([active]) {
-  @apply outline-none;
-}
-
-:host(:focus:not([disabled])) .day {
-  @apply focus-outset;
-  box-shadow: 0 0 0 2px var(--calcite-ui-foreground-1);
-}
-
-:host([selected]) .day {
-  @apply font-medium;
-  background-color: var(--calcite-ui-brand) !important;
-  color: var(--calcite-ui-foreground-1) !important;
-}
-
-:host([range][selected]) {
-  .day-wrapper {
-    background-color: var(--calcite-ui-foreground-current);
+:host(:focus) {
+  &[active] {
```

Comment:
Please double check the generated CSS. This is not equivalent to the replaced styles: https://gist.github.com/jcfranco/ebe42f1b7e5a47b3c20d93bb71c2cfce